examples/asn.js

```diff
@@ -1,4 +1,4 @@
-
 var maxmind = require('maxmind');
 maxmind.init(__dirname + '/GeoIPASNum.dat');
-console.log(maxmind.getOrganization('66.6.44.4'));
+
+console.log('ASN: ', maxmind.getOrganization('66.6.44.4'));
```

Comment:
Done. I can squash the commits later if you want because now there are 5 commits, last 4 of which are with some minor changes. What do you say ?